tests/lib/nested.sh

```diff
@@ -485,10 +479,26 @@ start_nested_core_vm_unit(){
     PARAM_MONITOR="-monitor tcp:127.0.0.1:$MON_PORT,server,nowait"
     PARAM_USB="-usb"
     PARAM_CD="${PARAM_CD:-}"
+    PARAM_RANDOM="-object rng-random,id=rng0,filename=/dev/urandom -device virtio-rng-pci,rng=rng0"
+    PARAM_CPU=""
+    PARAM_TRACE="-d cpu_reset"
+    PARAM_LOG="-D $WORK_DIR/qemu.log"
+    PARAM_SERIAL="-serial file:${WORK_DIR}/serial.log"
+
+    # Set kvm attribute
+    ATTR_KVM=""
+    if [ "$ENABLE_KVM" = "true" ]; then
+        ATTR_KVM=",accel=kvm"
+        # CPU can be defined just when kvm is enabled
+        PARAM_CPU="-cpu host"
+        # Increase the number of cpus used once the issue related to kvm and ovmf is fixed
+        # https://bugs.launchpad.net/ubuntu/+source/kvm/+bug/1872803
+        PARAM_SMP="-smp 1"
```

Comment:
I'm not sure we should limit the run to 1 CPU now that we selectively disable KVM. I suspect that if the problem was indeed with nested kvm emulation, it would just show up more frequently with more CPUs, so limiting to one only made the issue lest frequent.

Reply:
@bboozzoo, thanks for the review, I understand the idea but something that used to happen when we used 2 cpus is that the boot process never finished because of the reboots. And the idea is to still use kvm for local testing. Perhaps what we can do is to allow to pass a parameter with the number of the cpus?